templates/compositions/ec-component-language-list/language-list-selector.html.twig

```diff
@@ -71,7 +87,10 @@
   {{- _label -}}
  </a>
 {% include '@ecl-twig/language-list/language-list' with {
+  eu_category: eu_category,
```

Comment:
If the category vars don't exist yet, what happens here? Shouldn;t we initialize the variable?

Reply:
Twig takes care of that, no need initialising vars.